lib/neo4j/shared/type_converters.rb

```diff
@@ -271,6 +271,10 @@ class EnumConverter
       def initialize(enum_keys, options)
         @enum_keys = enum_keys
         @options = options
+
+        return unless @options[:case_sensitive].nil?
+
+        @options[:case_sensitive] = Neo4j::Config.enums_case_sensitive
```

Comment:
Instead of these two lines, what about just:

```ruby
@options[:case_sensitive] ||= Neo4j::Config.enums_case_sensitive
```

Reply:
This doesn't work because if `@options[:case_sensitive] == false` and `Neo4j::Config.enums_case_sensitive == true`, then `@options[:case_sensitive]` will be set to `true` when it should remain false (i.e. local settings should always take precedence over global settings). As it stands, global settings only kick in if `@options[:case_sensitive].nil?`.

Reply:
Ah, right, good point ;)